app/jobs/create_uphold_cards_job.rb

```diff
@@ -9,8 +9,17 @@ def perform(uphold_connection_id:)
       return
     end
 
+    card = nil
+
     # Search for an existing card
-    card = uphold_connection.uphold_client.card.where(uphold_connection: uphold_connection)&.first
+    cards = uphold_connection.uphold_client.card.where(uphold_connection: uphold_connection)
```

Comment:
Shouldn't there be a currency match too?

Reply:
In this case it uses the currency that is on the `uphold_connection` object, it's not very clear form the interface. Should I go ahead and pass the currency in?